### PR TITLE
Handler clean up + edge case protection

### PR DESCRIPTION
**Pull request by Brady76**
zzzzzzzzzzzzzzz

### CODE REVIEW

File: handlers/base_handler.py [hunk 1]
```diff
@@ -9,10 +9,14 @@ class BaseHandler(tornado.web.RequestHandler):
 
     def render(self, template_name, **kwargs):
         raw_data = kwargs.get('raw_data', '')
+        logging.info(raw_data)
```
Comment (untra) [inline handlers/base_handler.py]: delete this

File: handlers/base_handler.py [hunk 2]
```diff
@@ -21,11 +25,19 @@ def color(self):
     def keywords_string(self, raw_data):
         return """cufcq,university,colorado,faculty,course,instructor,fcq,grade,department,database"""
 
+    def noneCheck(self, raw_data):
+        if(raw_data != None):
```
Comment (untra) [inline handlers/base_handler.py]: if raw_data is not None:

Comment (untra) [inline handlers/base_handler.py]: `"default" if x is None else x`
is a one line ifelse

**Comment by Brady76** [comment]
@untra good now?

**Pull request merged**